pkg/index/validation/validate.go

```diff
@@ -110,15 +110,33 @@ func validatePlatform(p index.Platform) error {
 	if p.Bin == "" {
 		return errors.New("bin has to be set")
 	}
-	if len(p.Files) == 0 {
-		return errors.New("can't have a plugin without specifying file operations")
+	if err := validateFiles(p.Files); err != nil {
+		return errors.Wrap(err, "files is invalid")
 	}
 	if err := validateSelector(p.Selector); err != nil {
 		return errors.Wrap(err, "invalid platform selector")
 	}
 	return nil
 }
 
+func validateFiles(fops []index.FileOperation) error {
+	if fops == nil {
+		return nil
+	}
+	if len(fops) == 0 {
+		return errors.New("files has to be unspecified or non-empty")
+	}
+	for _, op := range fops {
+		if op.From == "" {
+			return errors.New("from field has to be set")
```

Comment:
can we say \"from\" and \"to\" (below) to distinguish field names from the rest of the error message. especially below it says `to ... to be set` so it's a bit confusing.

Reply:
I enclosed field names in backticks. To be consistent, I did the same for other error messages.